TestRawGridSync does not pass in my environment

### Conditions
* CuPy version: current master (82040653c6cb810cd0c7730480c38a8250f01fc0)
* OS/Platform: 
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.3 LTS"

$ uname -a
Linux localhost 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

* `cupy.show_config()`
```
$ python -c 'import cupy; cupy.show_config()'
CuPy Version          : 7.0.0
CUDA Root             : /usr/local/cuda
CUDA Build Version    : 10010
CUDA Driver Version   : 10010
CUDA Runtime Version  : 10010
cuBLAS Version        : 10201
cuFFT Version         : 10101
cuRAND Version        : 10101
cuSOLVER Version      : (10, 2, 0)
cuSPARSE Version      : 10300
NVRTC Version         : (10, 1)
cuDNN Build Version   : None
cuDNN Version         : None
NCCL Build Version    : 2104
NCCL Runtime Version  : (unknown)
```

### Problem description
```
$ pytest tests/cupy_tests/core_tests/test_raw.py -k 'TestRawGridSync'
...

>                   raise cex
E                   cupy.cuda.compiler.CompileException: `nvcc` command returns non-zero exit status. 
E                   command: ['nvcc', '-gencode=arch=compute_35,code=sm_35', '-I/data2/work/w/repos/cupy/cupy/core/include', '-I /data2/work/w/repos/cupy/cupy/core/include/cupy/_cuda/cuda-10.1', '-I /usr/local/cuda/include', '-ftz=true', '-rdc=true', '-Xcompiler', '-fPIC', '-shared', '-o', '/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.o', '/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu']
E                   return-code: 1
E                   stdout/stderr: 
E                   /tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu(7): error: namespace "cooperative_groups" has no member "grid_group"
E                   
E                   /tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu(7): error: namespace "cooperative_groups" has no member "this_grid"
E                   
E                   2 errors detected in the compilation of "/tmp/tmpxft_000068a5_00000000-8_047817c85c93e8b2fe6fd21b1aa97220_2.cpp1.ii".

cupy/cuda/compiler.py:215: CompileException
---------------------------------------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------------------------------------
NVCC compilation error: `nvcc` command returns non-zero exit status.
command: ['nvcc', '-gencode=arch=compute_35,code=sm_35', '-I/data2/work/w/repos/cupy/cupy/core/include', '-I /data2/work/w/repos/cupy/cupy/core/include/cupy/_cuda/cuda-10.1', '-I /usr/local/cuda/include', '-ftz=true', '-rdc=true', '-Xcompiler', '-fPIC', '-shared', '-o', '/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.o', '/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu']
return-code: 1
stdout/stderr:
/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu(7): error: namespace "cooperative_groups" has no member "grid_group"

/tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu(7): error: namespace "cooperative_groups" has no member "this_grid"

2 errors detected in the compilation of "/tmp/tmpxft_000068a5_00000000-8_047817c85c93e8b2fe6fd21b1aa97220_2.cpp1.ii".

-----
Name: /tmp/tmpr5hwghmg/047817c85c93e8b2fe6fd21b1aa97220_2.cu
Options: -I/data2/work/w/repos/cupy/cupy/core/include -I /data2/work/w/repos/cupy/cupy/core/include/cupy/_cuda/cuda-10.1 -I /usr/local/cuda/include -ftz=true -rdc=true -Xcompiler -fPIC -shared
CUDA source:
01
02 #include <cooperative_groups.h>
03
04 extern "C" __global__
05 void test_grid_sync(const float* x1, const float* x2, float* y) {
06     namespace cg = cooperative_groups;
07     cg::grid_group grid = cg::this_grid();
08     int size = gridDim.x * blockDim.x;
09     int tid = blockDim.x * blockIdx.x + threadIdx.x;
10     y[tid] = x1[tid];
11     cg::sync(grid);
12     y[size - tid - 1] += x2[size - tid - 1];
13 }
14
-----
======================================================================================== 6 failed, 26 deselected in 2.02s =========================================================================================
```

--- a/tests/cupy_tests/core_tests/test_raw.py
+++ b/tests/cupy_tests/core_tests/test_raw.py
@@ -540,14 +540,16 @@ void test_grid_sync(const float* x1, const float* x2, float* y) {
 @testing.parameterize(*testing.product({
     'n': [10, 100, 256]
 }))
+@unittest.skipUnless(
+    9000 <= cupy.cuda.runtime.runtimeGetVersion(),
+    'Requires CUDA 9.x or later')
+@unittest.skipUnless(
+    60 <= int(cupy.cuda.device.get_compute_capability()),
+    'Requires compute capability 6.0 or later')
 class TestRawGridSync(unittest.TestCase):
 
     def setUp(self):
         global _test_cache_dir
-        if cupy.cuda.runtime.runtimeGetVersion() < 9000:
-            pytest.skip('Requires CUDA 9.x or later')
-        if int(cupy.cuda.device.get_compute_capability()) <= 60:
-            pytest.skip('Requires compute capability 6.0 or later')
         _test_cache_dir = tempfile.mkdtemp()
         os.environ['CUPY_CACHE_DIR'] = _test_cache_dir
 

--- a/tests/cupy_tests/core_tests/test_raw.py
+++ b/tests/cupy_tests/core_tests/test_raw.py
@@ -537,9 +537,6 @@ void test_grid_sync(const float* x1, const float* x2, float* y) {
 '''
 
 
-@unittest.skipUnless(
-    9000 <= cupy.cuda.runtime.runtimeGetVersion(),
-    'Requires CUDA 9.x or later')
 @testing.parameterize(*testing.product({
     'n': [10, 100, 256]
 }))
@@ -547,6 +544,10 @@ class TestRawGridSync(unittest.TestCase):
 
     def setUp(self):
         global _test_cache_dir
+        if cupy.cuda.runtime.runtimeGetVersion() < 9000:
+            pytest.skip('Requires CUDA 9.x or later')
+        if int(cupy.cuda.device.get_compute_capability()) <= 60:
+            pytest.skip('Requires compute capability 6.0 or later')
         _test_cache_dir = tempfile.mkdtemp()
         os.environ['CUPY_CACHE_DIR'] = _test_cache_dir
 

